department-of-veterans-affairs/abd-vro
# Deploy scripts: remove unused Helm values

## What was the problem?


Deploy scripts set `images.mq.imageName` and `images.redis.imageName` Helm values which are not used. The images are specified in `helm-service-*/templates/*/deployment.yaml`

Setting unused values causes confusion.

Similarly, `helm-app/values.yaml` has unused values since separate `helm-service-*` folders have been set up.

## How does this fix it?[^1]

Remove unused Helm values

Bonus: [distinguish pg-isready container from db container](https://github.com/department-of-veterans-affairs/abd-vro/pull/974/commits/2121c867f10393c363e55028f72c49f0b59ecacb)

## How to test this PR
Deploy to LHDI.

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
